src/ngRoute/route.js

```diff
@@ -615,7 +615,13 @@ function $RouteProvider() {
             }
           }, function(error) {
             if (nextRoute == $route.current) {
-              $rootScope.$broadcast('$routeChangeError', nextRoute, lastRoute, error);
+              // If nothing's listening for a $routeChangeError event, we log it in the console
+              //  to aid in debugging
+              if ($rootScope.$$listenerCount['$routeChangeError'] === 0) {
+                console.log('No listeners found for $routeChangeError: '+ error);
```

Comment:
You should use `$log.debug()` (so warnngs can be disabled in production).
(And there's a missing space before `+` :smiley:)